README.md

```diff
@@ -147,6 +162,15 @@ Note: You can also pass a string in place of the `options` object as a shorthand
 
 Checks if given object `obj` is a *light-my-request* `Request` object.
 
+#### Method chaining
+
+There are following methods you can used as chaining:
+- `delete`, `get`, `head`, `options`, `patch`, `post`, `put`, `trace`. They will set the HTTP request method and also the request URL.
+- `body`, `headers`, `payload`, `query`. They can be used to set the request options object.
+And finally you need to call `end`. It has the signature `function (callback)`.
```

Comment:
end() can be invoked without a callback

Reply:
That's right. But I think that's not the way we should recommend users to do, so I would not mention it in the document.

If invoking `end()` without a callback, users should access the promise through methods like `then()` later on:
```js
inject(dispatch)
  .get('/')
  .end()
  .then(res => {})
```

It acts the same as:
```js
inject(dispatch)
  .get('/')
  .then(res => {})
```
So I think there is no need to call `end()`. We just provide a way that makes the application more flexible.